aiohttp/helpers.py

```diff
@@ -876,6 +878,14 @@ def set_cookie(
         if samesite is not None:
             c["samesite"] = samesite
 
+        if DEBUG:
+            cookie_length = len(c.output(header="")[1:])
+            if cookie_length > COOKIE_MAX_LENGTH:
+                warnings.warn(
+                    "The size of is too large, it might get ignored by the client.",
+                    UserWarning,
+                )
```

Comment:
You want `stacklevel=2` almost always:
```suggestion
                    stacklevel=2,
                )
```
(this will make the warning point at the calling code, meaning the user app, rather than aiohttp internals)

Reply:
Noted, Thanks for the tip.